src/db-service.ts

```diff
@@ -133,6 +133,10 @@ export class RecycledItemService {
     itemPaths: string[],
     transactionHandler: TrxHandler,
   ): Promise<Pick<Item, 'path'>[]> {
+    if (!itemPaths || !itemPaths.length) {
```

Comment:
We may want to take the time to set [`strictNullChecks`](https://www.typescriptlang.org/tsconfig#strictNullChecks) to `true` in the tsconfig.json at some point to avoid those kinds of existence checks.